app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -54,7 +55,7 @@
         <% end %>
       <% else %>
         <%= tag.legend class: legend_classes do %>
-          <%= tag.h2 heading, class: "govuk-fieldset__heading" %>
+          <%= content_tag(shared_helper.get_heading_level, heading, class: "govuk-fieldset__heading" ) %>
```

Comment:
Nitpick: there's a space inside the closing `)` but none after the opening one. I'd remove it.